webpack.config.node.js

```diff
@@ -81,7 +82,9 @@ const webpackConfig = {
 					plugins: [ [
 						path.join( __dirname, 'server', 'bundler', 'babel', 'babel-plugin-transform-wpcalypso-async' ),
 						{ async: false }
-					] ]
+					] ],
+					cacheDirectory: path.join( __dirname, './build/.babel-cache' ),
```

Comment:
> does path.join handle fixing embedded slashes in paths for windows?

yes.

https://shapeshed.com/writing-cross-platform-node/#use-path-join-to-join-folder-names